infra/core/ai/cognitiveservices.bicep

```diff
@@ -0,0 +1,42 @@
+metadata description = 'Creates an Azure Cognitive Services instance.'
```

Comment:
Is infra/core the same as azure-dev/infra/core?  Any changes needed for this app?

Reply:
So far they are consistent and we haven't changed anything. There may be changes later, please refer to my [comment](https://github.com/jongio/chat-with-your-data-solution-accelerator/pull/2#discussion_r1392265065) here.